modules/gdscript/gdscript_utility_functions.cpp

```diff
@@ -619,8 +619,8 @@ StringName GDScriptUtilityFunctions::get_function_return_class(const StringName
 Variant::Type GDScriptUtilityFunctions::get_function_argument_type(const StringName &p_function, int p_arg) {
 	GDScriptUtilityFunctionInfo *info = utility_function_table.lookup_ptr(p_function);
 	ERR_FAIL_NULL_V(info, Variant::NIL);
-	ERR_FAIL_COND_V(p_arg >= info->info.arguments.size(), Variant::NIL);
-	return info->info.arguments.get(p_arg).type;
+	ERR_FAIL_COND_V(p_arg >= (int)info->info.arguments.size(), Variant::NIL);
```

Comment:
Would `ERR_FAIL_INDEX_V` be appropriate here?

Reply:
Should be, will test!